api/test/unit/models/test_media_report.py

```diff
@@ -41,7 +40,7 @@ def test_pending_reports_have_no_subreport_models(
     media = media_type_config.model_factory.create()
     report = media_type_config.report_factory.create(media_obj=media, reason=reason)
 
-    assert report.status == PENDING
+    assert report.decision is None
```

Comment:
I suppose this can also be `assert report.is_pending`, to be more directly relevant to the previous expression.

Reply:
Good observation! Changes applied.

Reply:
Looks like this might have gotten mixed up in commits? Left a comment with suggested change.